lib/helpers.js

```diff
@@ -11,5 +11,5 @@ function shouldIndentChild(parent, child, opts) {
 
   var left = child.left;
```

Comment:
lint complained about this variable not being needed anymore, otherwise looks good to me! thanks a lot for the bug report and for fixing it so quickly!